android/app/webviewAssets.gradle

```diff
@@ -7,6 +10,45 @@
  * more information.
  */
 
+/**
+ * Windows support functions.
+ *
+ * On Windows, we have three different possible setups:
+ *   1. Windows Gradle + MSys Git's bash.
```

Comment:
Naming nit: "MSys Git" doesn't appear to be quite the name of anything.

There was an "msysGit": https://github.com/msysgit/msysgit whose name often got regularized to "msysgit". Looks like that's a few years obsolete, though. The new stack includes things named "Git for Windows"; "Git BASH"; "the Git SDK"; and ["MSYS2" or "msys2"](http://www.msys2.org/).

Does the "msys"/"MSYS" name ever show up in actually interacting with this stack (as far as you've seen)? Or are you just remembering, like me, the days when "msysgit" was the way people got Git on Windows?

I've mainly been referring to this GNU-like environment as "Git Bash", just regularizing the capitalization of "Git BASH" which seems to be what it calls itself for users.

I've seen enough references over the years to ~KSplice~ (the actual name was and is Ksplice) that in any case I'd like to not add internal capitals to other people's names. :wink:

Reply:
MSYS is thus named on [its official page](https://www.mingw.org/wiki/MSYS), as is [msys2](https://www.msys2.org/). "MSys" is all my fault.

>  Or are you just remembering, like me, the days when "msysgit" was the way people got Git on Windows?

Mostly this.